tools/schema_tools/prep_opsim.py

```diff
@@ -101,67 +138,214 @@ def offsetHex():
             vertex_count += 1
     return offsets
 
-def add_dither(database, simname, overwrite=True):
-    ## Add the Krughoff-Jones dithering pattern to the opsim output
-    ## adds three columns (ditheredRA, ditheredDec, and vertex) and indexes
-    #connect to the database
+def offsetRandom(noffsets, inHex=True):
+    """ 
+    Returns translational random dither offsets in radians.
+    
+    Required input
+    --------------
+    * noffsets: int: number of offsets needed.
+    
+    Optional input
+    --------------
+    * inHex: bool: set to False if dont want to restrict the dithers to within
+                   the hexagon inscribing the FOV. Default: True
+                  
+    """
+    import numpy as np
+    from lsst.sims.maf.stackers import inHexagon
```

Comment:
Oh shoot ... this adds a dependency on MAF to Opsim. 
We've been trying to avoid those. Possible solution is to just add that function here. Another one would be to just bite the bullet and make opsim depend on maf. (@danielfs and @mareuter )